web/pipeline/utils.py

```diff
@@ -550,3 +550,14 @@ def serialize_community_search_names(communities):
 
 def get_pct_field_as_decimal(field):
     return field / 100 if field else 0
+
+
+def serialize_communities_for_regional_districts(regional_districts):
+    return {
+        regional_district.name: [
+            {
+                "community": community.place_name,
+                "locations": community.location_set.count()
```

Comment:
would a name like num_locations make it clear this is a count, not a location_id?

Reply:
yeah I agree, although I'm not entirely sure it's supposed to be a count. @byshine can you confirm?